app/src/main/java/com/quran/labs/androidquran/presenter/translation/InlineTranslationPresenter.kt

```diff
@@ -38,11 +38,11 @@ class InlineTranslationPresenter @Inject constructor(
       .launchIn(scope)
   }
 
-  suspend fun refresh(verseRange: VerseRange) {
+  suspend fun refresh(verseRange: VerseRange, ayahHasBeenChanged: Boolean) {
```

Comment:
if we keep track of the last verse range within `InlineTranslationPresenter`, we can avoid having to update all the callers to pass in whether or not the ayah has changed - can we do that instead? i.e.:

```kotlin
private var lastVerseRange: VerseRange? = null

suspend fun refresh(verseRange: verseRange) {
  val ayahHasBeenChanged = verseRange == lastVerseRange
  // existing logic here
  lastVerseRange = verseRange
}
```

Reply:
Looks good! I changed the PR (forced pushed to emit the old commit)